Editor/Resources/EditorWindow/common.uss

```diff
@@ -289,7 +289,7 @@ Image#ExternalLinkIcon {
 }
 
 .text-input--fitted {
-    flex-shrink: 1;
+    flex-grow: 0;
```

Comment:
Does the [original suggestion](https://github.com/aws/amazon-gamelift-plugin-unity/pull/110/files?file-filters%5B%5D=.uss&file-filters%5B%5D=.uxml&show-viewed-files=true#r1347806161) for using `fit-content` not work? That's more natural and isn't prone to break if we change the parents' styling like what happened here